Automattic/wp-calypso
# Plans 2023: Migrate p2 plans to new pricing grid

Related to https://github.com/Automattic/wp-calypso/issues/77652

## Proposed Changes

* For the p2 flow plans page, replace the outdated pricing page with the 2023 pricing page `PlansFeaturesMain` component

**Note:**

We will be removing outdated references ( examples below ) to P2 plans in a separate PR

https://github.com/Automattic/wp-calypso/blob/ba8c3ff726444513308b8c91b740fbdee9df818c/client/my-sites/plan-features/actions.jsx#L67

https://github.com/Automattic/wp-calypso/blob/ba8c3ff726444513308b8c91b740fbdee9df818c/client/my-sites/plan-features/header.jsx#L42-L44, etc. )

We're also focusing less on perfecting the design and more on removing custom layout and styling of the previous p2 workspaces plans page. This is because the feature is in maintenance mode. Context here p9lV3a-6EY-p2

## Screenshots

### Before

<img width="857" alt="Screenshot 2023-10-13 at 8 40 10 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/9c1be856-3d11-4085-9b55-4ff98cde5d8a">

### After

<img width="1174" alt="Screenshot 2023-10-14 at 12 22 34 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/b22195ce-0c3b-4d00-b49d-eef1c8c0e6f6">

<img width="992" alt="Screenshot 2023-10-14 at 12 28 36 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/de9fab7f-1e63-4f7c-b838-ba31199ff474">

## Testing Instructions



* Create a new p2 site by visiting calypso.localhost:3000/start/p2 and following onboarding instructions
* Note the site slug that's created for your new p2 workspace
* Navigate directly to calypso.localhost:3000/plans/{P2_WORKSPACE_SITE_SLUG}
* Verify that the P2+ plan can still be purchased
* Verify that the "Your plan" badge highlights the correct plan
* Verify that, when the current plan is the p2 free plan, that the "Manage add-ons" button redirects to the /add-ons page in Calypso admin
* Verify that everything behaves as expected in the mobile view

## Pre-merge Checklist
